tests/test_material_files.py

```diff
@@ -5,15 +5,12 @@
 
 @pytest.fixture
 def LCs():
-    lattice_constants = {
-        'a' : 1,
-        'b' : 1,
-        'c' : 1
-    }
+    lattice_constants = {"a" : 1, "b" : 1, "c" : 1}
     return lattice_constants
 
 def test_number_density_bounds():
-    assert random_number_density((100000, 100000), LCs) == 100000
+    assert random_number_density((100000, 100000), LCs()) == 100000
 
 def test_two_site_constraint():
-    assert random_number_density((0, 1), LCs) == 2 and random_number_density((0, 2), LCs) == 2
+    assert random_number_density((1, 2), LCs()) == 2
+
```

Comment:
Ok, so I think this simple test is fine, but you should run it ~ 10 times so there is a < 1/1000 chance of it giving you a false positive. So something like 
tests = [ random_number_density((1, 2), LCs()) for _ in range(0,10)]
assert tests == [2]*10